com.github.reduz.ChibiTracker.appdata.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.reduz.ChibiTracker</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <name>ChibiTracker</name>
+  <summary>ChibiTracker is a portable IT (Impulse Tracker) clone</summary>
+
+  <description>
+    <p>
+      ChibiTracker is a portable IT (Impulse Tracker) clone. It is not a 100% clone though, as it adds more features, such as chorus and reverb, stereo samples, an advanced sample editor and more. The user interface also presents some changes, as it can be custom themed and resized.
+    </p>
+    <p>
+      ChibiTracker is Open Source Software under the GNU Public License, and it can be run under Windows, Mac-OS, Linux, FreeBSD, BeOS, and more to come.
+    </p>
+    <p>
+      For those who haven't used trackers before, ChibiTracker is a small, compact music composing application that is easy to learn and powerful enough to sound good.
+    </p>
+  </description>
+
+​  <launchable type="desktop-id">com.github.reduz.ChibiTracker.desktop</launchable>
+
+  <categories>
```

Comment:
Not needed, should be put on the desktop file instead.

Reply:
Is it really unneeded? I though Gnome Software used it or something.

Reply:
GNOME Software (and the flathub website) absolutely use it - but the recommended way to do this for Flathub is to add the categories to the desktop file (some desktop environments can only read them from there). They're then automatically copied into the appdata file during app compilation. So you don't need them in both places.

Reply:
Ah, I thought you were talking about the <launchable> tag since it was the most visible on the quoted code. Sure, I will remove the categories tag soon.